content/posts/data-protection-in-research-practice-–-a-tutorial/index.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Data Protection in Research Practice - A Tutorial
+lang: en
+date: 2021-11-19T06:59:53.616Z
```

Comment:
@VickyGarnett make sure you change the date to today's date

Reply:
Done :)